header.php

```diff
@@ -19,40 +19,52 @@
 </head>
 
 <body <?php body_class(); ?>>
-	<a class="skip-link screen-reader-text" href="#content"><?php esc_html_e( 'Skip to content', 'ampconf' ); ?></a>
 
-	<header class="wrap wrap--header wrap--header__menu-hidden" [class]="mobileMenu ? 'wrap wrap--header' : 'wrap wrap--header wrap--header__menu-hidden'">
+	<div id="page" class="" [class]="ampConf.mobileMenu ? 'no-scroll' : ''">
```

Comment:
If this `div` is only here because of the need to toggle its class, I've raised this with the AMP project where we should be able to toggle a class on an element that already has multiple classes. See https://github.com/ampproject/amphtml/issues/1032#issuecomment-362716322

Reply:
@westonruter The `div#page` is here to prevent page scrolling when the menu is opened on mobile. I tried to apply `[class]` syntax on the `body` tag, but it didn't work.

Reply:
Yes, it is a known limitation with AMP currently.